docker/nginx/api.nginx.conf

```diff
@@ -5,6 +5,21 @@ events { worker_connections 1024; }
 http {
   server {
     listen 9000;
+
+    # Ensure that requests sent over HTTP are redirected to HTTPS.
+    # http://scottwb.com/blog/2013/10/28/always-on-https-with-nginx-behind-an-elb/
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host $http_host;
+    proxy_redirect off;
+    proxy_next_upstream error;
+
+    if ($http_x_forwarded_proto != "https") {
+      rewrite ^(.*)$ https://api.wellcomecollection.org$1 permanent;
+    }
+
+    add_header Strict-Transport-Security "max-age=31536000; includeSubDomains;";
```

Comment:
[We're doing it here](https://github.com/wellcometrust/wellcomecollection.org/blob/74689a702475098683eed84d66fc073eae77eb91/server/middleware/enforce-ssl.js).
I'd like to move it to nginx so nginx is doing all things http.